SITnett/templates/meny.html

```diff
@@ -13,23 +13,31 @@
 
 	<status id="status">
 		<h1>
-		intern
+		{% if user.is_authenticated %}
+		<a href="{% url 'logout' %}">{{user.get_username}}</a>
+		{% else %}
+		<a href="{% url 'login' %}">ekstern</a>
 		<a href="{% url 'opptak' %}">Søk SIT</a>
+		{% endif %}
 		</h1>
 	</status>
 
 	<menu id="menu">
 		<h2>
 		<a href="{% url 'hoved' %}"><img src="{% static 'assets/sit_logo_no_subtitle.svg' %}" height=60></a>
+		{% if not user.is_authenticated %}
 		<a href="{% url 'info' %}">Om SIT</a>
+		{% endif %}
```

Comment:
Om SIT kan vel også være interessant for innloggede medlemmer? Spesielt nye.

Reply:
Tja. Tanken min var at 'Arkiv' erstatter 'Om SIT' for medlemmer. På 'Om SIT' ser jeg for meg at det står en kort presentasjon av hva SIT er -- noe aktive medlemmer sannsynligvis vet allerede. For nye medlemmer vil det være mer interessant å lese på 'Uttrykk' for å lære seg sjargongen, og 'Verv' for å finne informasjon om hvert verv.

Det er imidlertid ingen sak å endre hvem som får opp den knappen, hvis dere fortsatt mener den burde være synlig for medlemmer også.